src/rise-twitter.js

```diff
@@ -113,19 +115,23 @@ export default class RiseTwitter extends HTMLElement {
         this._play();
       } else {
         console.log('_handlePlay NOT IsAuthorized');
+        if (this.messaging.isConnected()) {
```

Comment:
@rodrigopavezi As discussed, there's some risk with this type of communication. Should be much more stable once we move away from modules.